shared/uavobjectdefinition/flightbatterysettings.xml

```diff
@@ -6,6 +6,9 @@
         <field name="VoltagePin" units="" type="enum" elements="1" options="ADC0,ADC1,ADC2,ADC3,ADC4,ADC5,ADC6,ADC7,ADC8,NONE" defaultvalue="ADC1"/>
 
         <field name="NbCells" units=""  type="uint8"  elements="1" defaultvalue="3"/>
+	<field name="MaxCellVoltage" units="V" type="float" elements="1" defaultvalue="0">

```

Comment:
Nice, but the indentation is now broken :P (surrounding lines use spaces, yours use tabs).